src/js/TuningSliders.js

```diff
@@ -177,37 +181,74 @@ TuningSliders.resetPidSliders = function() {
     }
 
     this.calculateNewPids();
-    this.updatePidSlidersDisplay();
 };
 
 TuningSliders.resetGyroFilterSlider = function() {
-    if (!this.cachedGyroSliderValues) {
+    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
```

Comment:
Same as above - there are at least 4 ways that each version check can be formulated (using `lt`, `lte`, `gt`, `gte`). Using a mix of all of these to check the same condition in the same file will make this hard to maintain.

Reply:
made all the same